db/seeds/mock_offers.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
wow michelle, where did you get the format from? 
I was thinking we had to drive our CHASS development image to the point of exporting offers!